libs/dh/market-participant/data-access-api/src/lib/dh-market-participant-gridarea-data-access-api.store.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Injectable } from '@angular/core';
+import { ComponentStore, tapResponse } from '@ngrx/component-store';
+import { MarketParticipantGridAreaHttp } from '@energinet-datahub/dh/shared/domain';
+import { Observable, switchMap, tap } from 'rxjs';
+import { HttpErrorResponse } from '@angular/common/http';
+import { parseErrorResponse } from './dh-market-participant-error-handling';
+
+interface MarketParticipantGridAreaState {
+  isLoading: boolean;
+
+  // Changes
+  changes: GridAreaChanges;
```

Comment:
suggestion: this property is not used anywhere in the file (no data is saved to it). The data coming from the modal is sent directly to the endpoint. So the property can be removed.